frontend/src/pages/Footer/_Footer.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./_Footer.scss";
+import { Button } from "../../components/Buttons/Buttons";
+
+const Footer = ({ menu, logoDesktop, logoMobile, logoFooter }) => {
+  const Logo = () => {
+    return (
+      <div className="inner-one">
+        <a href="/">
+          <img className="logo-desktop-footer" src={logoDesktop} />
+          <img className="logo-mobile-footer" src={logoMobile} />
+        </a>
+      </div>
+    );
+  };
+
+  return (
+    <div className="footer-nav flex-container ">
+      <Logo />
+      <div className="footer-menu">
+        {menu.map((item, idx) => {
```

Comment:
A nitpick: Let me know if you prefer idx over index, since "index" is used in the headerNav and we should commit to one. If you like idx, it'll help with consistency if it is changed in the headerNav.

Reply:
I reverted back to index.